tests/openvino/test_exporters_cli.py

```diff
@@ -86,31 +86,31 @@ class OVCLIExportTestCase(unittest.TestCase):
     )
 
     TEST_4BIT_CONFIGURATONS = [
-        ("text-generation-with-past", "opt125m", "int4_sym_g128", 4, 144),
+        ("text-generation-with-past", "opt125m", "int4_sym_g128", 4, 72),
```

Comment:
Starting form NNCF 2.12, symmetric compression of weights is performed without zero point. This halves the number of reference int4 constants, because for each symmetrically compressed weight there used to be 2 low precision constants: one for the weight and one for the zero point.

cc @l-bat